Source/Charts/Utils/ChartUtils.swift

```diff
@@ -16,9 +16,26 @@ import CoreGraphics
     import UIKit
 #endif
 
+extension FloatingPoint {
+    var deg2rad: Self {
+        return self * .pi / 180
+    }
+
+    var rad2deg: Self {
+        return self * 180 / .pi
+    }
+
+    /// - returns: An angle between 0.0 < 360.0 (not less than zero, less than 360)
+    /// NOTE: Value must be in degrees
+    var normalizedAngle: Self {
+        let angle = truncatingRemainder(dividingBy: 360)
+        return (sign == .minus) ? angle + 360 : angle
```

Comment:
the order is flipped. I need to refresh my mind first to see if they are equivalent. Or you already did that?

Reply:
They’re equivalent.

Reply:
made some stupid equations :P